spec/services/create_order_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe CreateOrder, type: :service do
+  describe 'CreateOrder.call' do
```

Comment:
```suggestion
  describe 'call' do
```
`#` é para indicar que o método é publico
`.` para indicar que é estático